File: kafka3/src/main/scala/com/stulsoft/kafka3/AutoOffsetRest.scala
/*
 * Copyright (c) 2017. Yuriy Stul
 */

package com.stulsoft.kafka3

/**
  * @author Yuriy Stul
  */
object AutoOffsetRest extends Enumeration {
  type AutoOffsetRest = Value
  val Earliest: Value = Value("earliest")
  val Latest: Value = Value("latest")
  val None: Value = Value("none")
}

File: kafkaj/src/main/java/com/stulsoft/kafkaj/AutoOffsetRest.java
package com.stulsoft.kafkaj;

/**
 * @author Yuriy Stul.
 */
public enum AutoOffsetRest {
    Earliest("earliest"),
    Latest("latest"),
    None("none");
    private final String value;

    AutoOffsetRest(final String value) {
        this.value = value;
    }


    @Override
    public String toString() {
        return value;
    }
}
